client/src/components/Common/Button/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, ConfigProvider } from 'antd';
+import 'antd/dist/antd.css';
+
+const ButtonComponent = ({
+  Content,
+  disabled,
```

Comment:
what is the deferent between the block and disable?

Reply:
the disable props disable the button;
the block props make the button take the full width of the container.